README.md

```diff
@@ -47,184 +45,208 @@ http://ryanstutorials.net/linuxtutorial/
-
-**3. Configure hostname**
-
-Make sure that your hostname is correctly configured in /etc/hosts.
-See (https://stackoverflow.com/questions/24797947/os-x-and-rabbitmq-error-epmd-error-for-host-xxx-address-cannot-connect-to-ho). If you are testing with VMs make please make sure to provide unique host names for each of the VM you are using. 
-
-Hostname should be resolvable to a valid ip when running on bridged mode. RabbitMQ checks for this during
-initial boot. Without this (for example, when running on a VM in NAT mode)
-RabbitMQ  start would fail with the error "unable to connect to empd (
-port 4369) on <hostname>." Note: RabbitMQ startup error would show up in syslog (/var/log/messages) file
-and not in RabbitMQ logs (/var/log/rabbitmq/rabbitmq@hostname.log)
-
-
-**4. Download VOLTTRON code from experimental branch**
+From version 6.0 VOLTTRON supports two message bus - ZMQ and RabbitMQ. Clone the appropriate
+branch of volttron code. The develop branch currently only supports ZMQ. The 6.0alpha branch
+is backward compatible and supports both ZMQ and RMQ.
 
 ```sh
-git clone -b rabbitmq-volttron https://github.com/VOLTTRON/volttron.git
-cd volttron
-python bootstrap.py --rabbitmq [optional install directory. defaults to
-<user_home>/rabbitmq_server]
+git clone https://github.com/VOLTTRON/volttron --branch <branch name>
 ```
 
-This will build the platform and create a virtual Python environment and
-dependencies for RabbitMQ. It also installs RabbitMQ server as the current user.
-If an install path is provided, path should exists and be writeable. RabbitMQ
-will be installed under <install dir>/rabbitmq_server-3.7.7 Rest of the
-documentation refers to the directory <install dir>/rabbitmq_server-3.7.7 as
-$RABBITMQ_HOME
-
-You can check if RabbitMQ server is installed by checking it's status. Please
-note, RABBITMQ_HOME environment variable can be set in ~/.bashrc. If doing so,
-it needs to be set to RabbitMQ installation directory (default path is
-<user_home>/rabbitmq_server/rabbitmq_server/rabbitmq_server-3.7.7)
+###3. Setup virtual environment
 
-```
-echo 'export RABBITMQ_HOME=$HOME/rabbitmq_server/rabbitmq_server-3.7.7'|sudo tee --append ~/.bashrc
-source ~/.bashrc
-```
-
-```
-$RABBITMQ_HOME/sbin/rabbitmqctl status
-```
-
-Activate the environment :
+####Steps for ZMQ
+Run the following command to install all required packages
 
 ```sh
-. env/bin/activate
+cd <volttron clone directory>
+python bootstrap.py
+source env/bin/activate
 ```
 
-**4. Create RabbitMQ setup for VOLTTRON :**
-```
-vcfg --rabbitmq single [optional path to rabbitmq_config.yml]
-```
+Proceed to step 4. 
 
-Refer to examples/configurations/rabbitmq/rabbitmq_config.yml for a sample configuration file.
-At a minimum you would need to provide the host name and a unique common-name
-(under certificate-data) in the configuration file. Note. common-name must be
-unique and the general conventions is to use  <voltttron instance name>-root-ca.
+####Steps for RabbitMQ
```

Comment:
Why not move this entire section under setup/RabbitMQ folder? In this way they are all contained within that folder.

Reply:
Because I wanted to have at least the basic setup steps available in the README page and point to readthedocs for more detailed instruction. This is similar to what we do for ZMQ